### PR TITLE
Updated Bengali Python course

### PR DESCRIPTION
**Pull request by apu52**
## What does this PR do?
Add resource(s) 

### Description
Hello Sir,
I have added 2 very useful python courses in Bengali .It is useful for all the beginners as well as for the python developers.
### Why is this valuable (or not)?
This is a Valuable course.
### How do we know it's really free?
As the whole playlist is available on YouTube. It is totally free.
### For book lists, is it a book? For course lists, is it a course? etc.
It is a course list.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -280,6 +280,8 @@
 * [Python Bangla Tutorial & Data Science Full Course](https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT) - Study Mart
 * [Python Bangla Tutorial 2023](https://www.youtube.com/playlist?list=PLNMnAEqLBwmpR8JDBOEl0jrzmH1vPnO7v) - Hablu Programmer
 * [Python Bangla Tutorials for Beginners](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rz5XqMsTk41_j15_6682BN) - Anisul Islam
+* [Python For Beginners | Bangla Tutorials](https://www.youtube.com/playlist?list=PLvr0Ht-XkB_0V-mjAYlfgk-3VRmFarlzC) - Learn With Tawhid
```
Comment (sohansai) [inline courses/free-courses-bn.md:283]:
```suggestion
* [Python For Beginners - Bangla Tutorials](https://www.youtube.com/playlist?list=PLvr0Ht-XkB_0V-mjAYlfgk-3VRmFarlzC) - Learn With Tawhid
```

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -280,6 +280,8 @@
...
+* [Python Full Course Bangla ( Learn Python A - Z )](https://www.youtube.com/playlist?list=PLF-F70WLa6yP0gIAowyaluE85ZBBMB6en) - Artificial Neuron
```
Comment (sohansai) [inline courses/free-courses-bn.md:284]:
```suggestion
* [Python Full Course Bangla (Learn Python A - Z)](https://www.youtube.com/playlist?list=PLF-F70WLa6yP0gIAowyaluE85ZBBMB6en) - Artificial Neuron
```

**Comment by apu52** [comment]
@sohansai I have made the changes. Can you please check those

**Comment by apu52** [comment]
@eshellman Please take a review of my PR & please merge it.

**Pull request merged**